notebooks/500__jupyterhub/elyra/run-pipelines-on-data-science-pipelines/hello-generic-world.pipeline

```diff
@@ -306,7 +334,16 @@
         "properties": {
           "name": "hello-generic-world",
           "runtime": "Data Science Pipelines",
-          "description": "A generic pipeline tutorial"
+          "description": "A generic pipeline tutorial",
+          "pipeline_defaults": {
+            "kubernetes_shared_mem_size": {},
+            "kubernetes_tolerations": [],
+            "kubernetes_pod_annotations": [],
+            "kubernetes_pod_labels": [],
+            "mounted_volumes": [],
+            "env_vars": [],
+            "kubernetes_secrets": []
+          }
```

Comment:
I think these fields should not be included

Reply:
Removed it :+1: